src/views/bankrupt.html

```diff
@@ -24,40 +24,78 @@ <h1 class="govuk-heading-l">
     <fieldset class="govuk-fieldset" role="group" aria-describedby="dob-hint">
       <legend class="govuk-fieldset__legend">
-        Date of birth
+        From Date of Birth
       </legend>
       <div id="dob-hint" class="govuk-hint">
-        For example, 31 3 1980
+        For example, 31 03 1980
       </div>
-      <div class="govuk-date-input" id="dob">
+      <div class="govuk-date-input" id="from-dob">
         <div class="govuk-date-input__item">
           <div class="govuk-form-group">
-            <label class="govuk-label govuk-date-input__label" for="dob-day">
+            <label class="govuk-label govuk-date-input__label" for="from-dob-day">
               Day
             </label>
-            <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="dob-dd"
-              name="dob-dd" autocomplete="bday-day" type="text" pattern="[0-9]*" inputmode="numeric">
+            <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="from-dob-dd"
+              name="from-dob-dd" autocomplete="bday-day" type="text" pattern="[0-9]*" inputmode="numeric">
           </div>
         </div>
         <div class="govuk-date-input__item">
           <div class="govuk-form-group">
-            <label class="govuk-label govuk-date-input__label" for="dob-mm">
+            <label class="govuk-label govuk-date-input__label" for="from-dob-mm">
               Month
             </label>
-            <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="dob-mm"
-              name="dob-mm" autocomplete="bday-month" type="text" pattern="[0-9]*" inputmode="numeric">
+            <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="from-dob-mm"
+              name="from-dob-mm" autocomplete="bday-month" type="text" pattern="[0-9]*" inputmode="numeric">
           </div>
         </div>
         <div class="govuk-date-input__item">
           <div class="govuk-form-group">
-            <label class="govuk-label govuk-date-input__label" for="dob-yyyy">
+            <label class="govuk-label govuk-date-input__label" for="from-dob-yyyy">
               Year
             </label>
-            <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="dob-yyyy"
-              name="dob-yyyy" autocomplete="bday-year" type="text" pattern="[0-9]*" inputmode="numeric">
+            <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="from-dob-yyyy"
+              name="from-dob-yyyy" autocomplete="bday-year" type="text" pattern="[0-9]*" inputmode="numeric">
+            </div>
           </div>
         </div>
-      </div>
+  </fieldset>
+      </div><div class="govuk-form-group">
+        <fieldset class="govuk-fieldset" role="group" aria-describedby="dob-hint">
+          <legend class="govuk-fieldset__legend">
+           To Date of Birth:
+          </legend>
+          <div id="dob-hint" class="govuk-hint">
```

Comment:
Thinking maybe renaming the `id` here as there's already a div with an id of `dob-hint` in the from date section. Rename them to be appropriate whether it's from or to dob?